packages/paste-core/components/disclosure/src/Disclosure.tsx

```diff
@@ -16,12 +16,20 @@ export interface DisclosureContextProps {
 
 export const DisclosureContext = React.createContext<DisclosureContextProps>({} as any);
 
+interface LooseObject {
```

Comment:
Nit, I'd be tempted to lose this interface and just declare the `[key: string]: any` as a property on the DisclosureStateReturn interface